docs/setup/git_setup.md

```diff
@@ -0,0 +1,17 @@
+# Github Setup
+
+If you haven't set up git before, follow this guide.
+
+## Configuring Username and Email
+
+The first thing you should do when you install Git is to set your user name and email address. This is important because every Git commit uses this information, and it’s immutably baked into the commits you start creating:
```

Comment:
The last sentence sounds a bit weird.
Maybe rephrase it such that it talks about authorship rather than "being baked into" the commit.